### PR TITLE
Second and third section

### PR DESCRIPTION
**Pull request by vsafonova**
-Created layout for the second and third sections, 
-add the image with the cat with pancakes,
-change the Chinese text in the second section into Japanese (Just translated English text into Japanese),
-delete fonts import

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -103,14 +80,55 @@ p {
 
 .bg_white {
   background-color: #fff;
+  display: flex;
```
Comment (elen-oz) [inline src/styles.css:86]: here

File: src/styles.css [hunk 2]
```diff
@@ -103,14 +80,55 @@ p {
...
+}
+
+.bg_white h3  {
+  font-size: 4rem;
+  width: 30%;
 }
 
 .bg_white p {
-  width: 40%;
+  width: 35%;
```
Comment (elen-oz) [inline src/styles.css:98]: here

File: src/styles.css [hunk 3]
```diff
@@ -103,14 +80,55 @@ p {
...
+  font-weight: 600;
+  font-size: 1rem;
+}
+
+.bg_white .paragraph-eng {
+  color: #b8b2b2;
```
Comment (elen-oz) [inline src/styles.css:104]: here

File: src/styles.css [hunk 4]
```diff
@@ -103,14 +80,55 @@ p {
...
+  font-size: 0.7rem;
 }
 
 .bg_orange {
   background-color: #ff6636;
```
Comment (elen-oz) [inline src/styles.css:106]: here

File: src/styles.css [hunk 5]
```diff
@@ -103,14 +80,55 @@ p {
...
+  background-color: rgb(246, 240, 210);
+  display: flex;
+  align-items: end;
+  padding-bottom: 3rem;
+  justify-content: center;
+  padding-inline: 5rem;
+
+}
+
+.bg_orange img {
```
Comment (elen-oz) [inline src/styles.css]: here

**Review by elen-oz — CHANGES_REQUESTED** [review]

File: src/styles.css (hunk 1 above)
Comment (Quantinx) [reply src/styles.css:86]: This may cause issues for me, will check

**Review by Quantinx — APPROVED** [review]
Have compensated for the changes in my own classes

**Review by elen-oz — APPROVED** [review]

**Pull request merged**